mypy.ini

```diff
@@ -276,9 +273,6 @@ ignore_errors = True
 [mypy-torch.nn.cpp]
 ignore_errors = True
 
-[mypy-torch.nn.functional]
-ignore_errors = True
```

Comment:
Was this already passing, or did some of the changes in this PR make it pass?

Reply:
I was already passing